auditlog/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Audit Log",
-    "version": "14.0.1.2.0",
+    "version": "14.0.1.3.0",
```

Comment:
Because no versioned migration script is present in this PR, you can leave the version number increment to the OCA bot.